src/gluonts/mx/distribution/empirical_distribution.py

```diff
@@ -0,0 +1,258 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from typing import Optional, Tuple
+
+import numpy as np
+import mxnet as mx
+
+from gluonts.core.component import validated
+from gluonts.mx import Tensor
+
+from .distribution import Distribution, getF
+from .distribution_output import DistributionOutput, ArgProj
+
+
+class EmpiricalDistribution(Distribution):
+    r"""
+    A class representing empirical distribution.
+    By default, the target is assumed to be vector-valued, i.e., `event_shape` is 1. However, note that each dimension
```

Comment:
There should be an empty line after the short description (before starting a longer paragraph)

Reply:
Done.